django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -69,32 +74,55 @@ Requires core.js and SelectBox.js.
             filter_input.id = field_id + '_input';
 
             selector_available.appendChild(from_box);
-            const choose_all = quickElement('a', selector_available, gettext('Choose all'), 'title', interpolate(gettext('Click to choose all %s at once.'), [field_name]), 'href', '#', 'id', field_id + '_add_all_link');
-            choose_all.className = 'selector-chooseall';
+            const choose_all = quickElement(
+                'button',
+                selector_available,
+                interpolate(gettext('Choose all %s'), [field_name]),
+                'id', field_id + '_add_all_link',
+                'class', 'selector-chooseall'
+            );
 
             // <ul class="selector-chooser">
             const selector_chooser = quickElement('ul', selector_div);
             selector_chooser.className = 'selector-chooser';
-            const add_link = quickElement('a', quickElement('li', selector_chooser), gettext('Choose'), 'title', gettext('Choose'), 'href', '#', 'id', field_id + '_add_link');
-            add_link.className = 'selector-add';
-            const remove_link = quickElement('a', quickElement('li', selector_chooser), gettext('Remove'), 'title', gettext('Remove'), 'href', '#', 'id', field_id + '_remove_link');
-            remove_link.className = 'selector-remove';
+            const add_link = quickElement(
+                'a',
+                quickElement('li', selector_chooser),
+                interpolate(gettext('Choose selected %s'), [field_name]),
+                'href', '#',
+                'id', field_id + '_add_link',
+                'role', 'button',
+                'class', 'selector-add'
+            );
+            const remove_link = quickElement(
+                'a',
+                quickElement('li', selector_chooser),
+                interpolate(gettext('Remove selected chosen %s'), [field_name]),
+                'href', '#',
+                'id', field_id + '_remove_link',
+                'role', 'button',
+                'class', 'selector-remove'
+            );
```

Comment:
```suggestion
            const add_link = quickElement(
                'button',
                quickElement('li', selector_chooser),
                interpolate(gettext('Choose selected %s'), [field_name]),
                'id', field_id + '_add_link',
                'class', 'selector-add'
            );
            const remove_link = quickElement(
                'button',
                quickElement('li', selector_chooser),
                interpolate(gettext('Remove selected chosen %s'), [field_name]),
                'id', field_id + '_remove_link',
                'class', 'selector-remove'
            );
```
Can you also look into the style (and potentially javascript) changes required to get this to work?